cmd/experimental/kjobctl/pkg/cmd/create/create.go

```diff
@@ -47,33 +53,47 @@ const (
 	--parallelism 4 \
 	--completions 4 \ 
 	--request cpu=500m,ram=4Gi \
-	--localqueue my-local-queue-name`
+	--localqueue my-local-queue-name
+
+  # Create interactive 
+  kjobctl create interactive \ 
+	--profile my-application-profile  \
+	--pod-running-timeout 30s \
+	--rm`
 	profileFlagName     = "profile"
 	commandFlagName     = "cmd"
 	parallelismFlagName = "parallelism"
 	completionsFlagName = "completions"
 	requestFlagName     = "request"
 	localQueueFlagName  = "localqueue"
+	podRunningTimeout   = "pod-running-timeout"
 )
 
 var (
 	invalidApplicationProfileModeErr = errors.New("invalid application profile mode")
+	podRunningTimeoutDefault         = 1 * time.Minute
 )
 
 type CreateOptions struct {
+	exec.StreamOptions
 	PrintFlags *genericclioptions.PrintFlags
+	Config     *restclient.Config
+	Attach     attach.RemoteAttach
+	AttachFunc func(*CreateOptions, *corev1.Container, remotecommand.TerminalSizeQueue, *corev1.Pod) func() error
 
 	DryRunStrategy util.DryRunStrategy
 
 	Namespace   string
 	ProfileName string
 	ModeName    v1alpha1.ApplicationProfileMode
 
-	Command     []string
-	Parallelism *int32
-	Completions *int32
-	Requests    corev1.ResourceList
-	LocalQueue  string
+	Command              []string
+	Parallelism          *int32
+	Completions          *int32
+	Requests             corev1.ResourceList
+	LocalQueue           string
+	PodRunningTimeout    time.Duration
+	RemoveInteractivePod bool
```

Comment:
Can we check that this flags only for interactive mode?

Reply:
RemoveInteractivePod already is used only for interactive mode, but  PodRunningTimeout wasn't. I've updated the check of the latter to only apply it for interactive.

Reply:
Yes, but I think we need to send error if trying to use this flags with another mode.

Reply:
The create command has been refactored to have subcommands with their own flags.

Reply:
Cool. Thanks.